src/components/DeploymentDetails.vue

```diff
@@ -16,7 +16,8 @@
       <template #label>
         <div class="deployment-details__schedule-label">
           <span>Schedules</span>
-          <DeploymentToggle v-if="can.update.deployment && deployment.schedules.length" :deployment="deployment" @update="emit('update')" />
+          <!-- We need to display this toggle if the deployment is paused so a user can enabled it -->
+          <DeploymentToggle v-if="can.update.deployment && (deployment.schedules.length > 3 || deployment.paused)" :deployment="deployment" @update="emit('update')" />
```

Comment:
What's with the magic number 3 here?

Reply:
Discussed with the team generally when I demoed this initially -- mostly a matter of taste w/ regards to the space. Happy to put it in a config file somewhere if that's the correct pattern